src/libs/actions/Task.js

```diff
@@ -286,7 +286,7 @@ function editTaskAndNavigate(report, ownerEmail, title, description, assignee) {
     const editTaskReportAction = ReportUtils.buildOptimisticEditedTaskReportAction(ownerEmail);
 
     // Sometimes title is undefined, so we need to check for that, and we provide it to multiple functions
-    const reportName = title || report.reportName;
+    const reportName = title.trim() || report.reportName;
```

Comment:
Can't `title` be null here? Or do we have `|| report.reportName` here to handle when title is an empty string?

Reply:
Same in the 2 below.

Reply:
for `description` we have added `defaultValue=''`

Reply:
`title` is a mandatory field so might not be a null and we have added `defaultValue=""` for `description` field

Reply:
Oops! The comment above the code suggests that title can be undefined sometimes. I am not sure in what case is the title undefined.

Reply:
If it is `null` it will also fail. Anyway, if you are sure no one is passing that, then we can keep it.
Anyway I think we need change this to:
```suggestion
    const reportName = (title || report.reportName).trim();
```

Because otherwise we would not be trimming the reportName in case we are using it.

Reply:
changes applied to the title and description